### PR TITLE
python request

### PR DESCRIPTION
**Pull request by 11TK11**

### CODE REVIEW

File: src/kevinsanchez/python_request/RequestManager.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+"""
+class to manage requests.
+"""
+class RequestManager:
+
+    """
+    constructor for header.
+    """
+    def __init__(self):
+        self.base_url = 'https://todo.ly/api'
```
Comment (sperBruno) [inline src/kevinsanchez/python_request/RequestManager.py]: read from a properties or config file

File: config.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[Environment]
+url=https://todo.ly/api
+apiToken=2bb7223f8f6c429299ba7b7a744d0c7e
```
Comment (sperBruno) [inline config.properties]: remove data that can affect security

File: test/kevinsanchez/python_request/config.properties [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[Environment]
```
Comment (sperBruno) [inline test/kevinsanchez/python_request/config.properties]: Which config file are you using? you should use only one

**Review by 11TK11 — COMMENTED** [review]
delete information about user that is why there is a observation from travis

File: test/kevinsanchez/python_request/TestRequestManager.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+class TestRequestManager(unittest.TestCase):
+    """
+    set up for test.
+    """
+
+    def setUp(self):
+        self.request_manager = RequestManager()
+        import os
+        file = open((os.path.dirname(__file__))+'/schema-project.json')
+        schema_data = file.read()
```
Comment (sperBruno) [inline test/kevinsanchez/python_request/TestRequestManager.py]: create a reader util

**Comment by 11TK11** [comment]
worked with python -m unittest discover -s test/ -p "*.py" locally

**Pull request merged**